### ISSUE TITLE
EPIC - Develop classification model and evaluate performance

### ISSUE THREAD

**Issue by CorDamir** [CorDamir/mildew-alert#5]
Goal:

Design, train and evaluate a neural network that classifies cherry leaves as either healthy or infected

Acceptance criteria:
  - model accurately classifies at least 97% of the test set
